Singularity/Singularity/Sound/SoundManager.cs

```diff
@@ -0,0 +1,482 @@
+                {
+                    mAllSongs[levelName][(int) SoundPhase.Build] = song;
+                }
+
+                if (phase == "Battle")
+                {
+                    mAllSongs[levelName][(int) SoundPhase.Battle] = song;
+                }
+            }
+
+            foreach (string s in Directory.GetFiles(@"Content\Sound\SFX", "*.xnb"))
+            {
+                string fullName = s.Substring(8);
+                int endPos = fullName.LastIndexOf(".", StringComparison.Ordinal);
+                fullName = fullName.Substring(0, endPos);
+                SoundEffect effect = contentManager.Load<SoundEffect>(fullName);
+                string effectName = fullName.Substring(fullName.LastIndexOf(@"\", StringComparison.Ordinal) + 1);
+                mEffects[effectName] = effect;
+                Console.WriteLine($"Loaded {effectName}");
+            }
+
+            foreach (string s in Directory.GetFiles(@"Content\Sound\UI", "*.xnb"))
+            {
+                string fullName = s.Substring(8);
+                int endPos = fullName.LastIndexOf(".", StringComparison.Ordinal);
+                fullName = fullName.Substring(0, endPos);
+                SoundEffect effect = contentManager.Load<SoundEffect>(fullName);
+                string effectName = fullName.Substring(fullName.LastIndexOf(@"\", StringComparison.Ordinal) + 1);
+                mEffects[effectName] = effect;
+                Console.WriteLine($"Loaded {effectName}");
+            }
+
+            //sSoundtrack = contentManager.Load<Song>("BGmusic");
+            sSoundtrack = mAllSongs["Tutorial"][(int) SoundPhase.Build];
+        }
+
+        #endregion
+
+        public static void PlaySoundTrack()
+        {
+            if (sSoundtrack == null)
+                return;
+
+            MediaPlayer.Play(sSoundtrack);
+        }
+
+
+        /// <summary>
+        /// Adds an instance of the sound effect with the given properties to the Dictionary containing all instances and plays it.
+        /// </summary>
+        /// <param name="name">The sound effect's name.</param>
+        /// <param name="x">x coordinate where the sound is supposed to be played.</param>
+        /// <param name="y">y coordinate where the sound is supposed to be played.</param>
+        /// <param name="volume">The supposed sound volume.</param>
+        /// <param name="pitch">The pitch modificator.</param>
+        /// <param name="isGlobal">Specifies whether the sound has a position.</param>
+        /// <param name="loop">Specifies whether the sound is supposed to loop.</param>
+        /// <param name="soundClass">Specifies the SoundClass e.g. Effect.</param>
+        /// <returns>The id of the instance if params are valid. -1 if the SoundClass is invalid. -2 if the effect doesn't exist.</returns>
+        public int PlaySound(string name, float x, float y, float volume, float pitch, bool isGlobal, bool loop, SoundClass soundClass)
```

Comment:
if the method for playing other sounds are static, should this also be made static?